engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/views/subscription-type.html

```diff
@@ -12,7 +12,7 @@
 </div>
 <div ng-if="subscription.virt_only === true && subscription.host">
   <span translate> Guests of </span>
-  <a href='/content_hosts/{{ subscription.host.id }}' translate>
+  <a href='/content_hosts/{{ subscription.host.id }}/info' translate>
```

Comment:
I know you're just updating what's there but do you mind changing this to use `ui-sref` instead of href please?

```
ui-sref="content-host.info({hostId: subscription.host.id })" 
```